sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1002,6 +1002,25 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
      if(debugprint)print*,'sample ',VarName,' =',hwp(isa,jsa)
       endif
 
+! lightning threat index 1
+      VarName='ltg1_max'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,ltg1_max(ista_2l,jsta_2l))
+     if(debugprint)print*,'sample ',VarName,' =',ltg1_max(isa,jsa)
+
+! lightning threat index 2
+      VarName='ltg2_max'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,ltg2_max(ista_2l,jsta_2l))
+     if(debugprint)print*,'sample ',VarName,' =',ltg2_max(isa,jsa)
+
+! lightning threat index 3
+     ltg3_max = 5
```

Comment:
@SamuelTrahanNOAA Why is ltg3_max assigned as 5?

Reply:
That was to convince myself it really is being read in. It's left over from debugging; I'll remove it.